samples/database-get-default-leader.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2024 Google LLC
+ * Copyright 2021 Google LLC
```

Comment:
Revert? We should have this as 2024.

Reply:
Since, there is no any change in the old sample. We do not need this update, if done, header-checks will get fail in presubmit